### PR TITLE
Link to the SIG interop tools terminology docs

### PR DESCRIPTION
**Pull request by afrittoli**
We discussed in the SIG interoperability that we would like to make
the tools terminology document more discoverable.

Adding the document introduction and link to the full document in
the learn section of the best practices web-site.

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: content/en/learn/tools-terminology/_index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: "Terminology Used by CI/CD Tools"
+linkTitle: "Tools Terminology"
+weight: 200
+description: >
+  By the SIG Interoperability - a catalog of tools terminology
+---
+
+# Introduction
+
+There are many ways to greet someone and as humans if we do not understand the word being used we have the ability to observe body language, process tone, and even touch. These many different natural inputs allow us as humans to establish shared vocabulary upon which we have been able to build successful components relevant to our way of living and social norms of interacting.
```
Comment (justinabrahms) [inline content/en/learn/tools-terminology/_index.md:12]:
```suggestion
There are many ways to greet someone. If we do not understand the words being used, we have the ability to observe body language, process tone, and even touch. These many different natural inputs allow us as humans to establish shared vocabulary relevant to our way of living and social norms of interacting.
```

File: content/en/learn/tools-terminology/_index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Unfortunately for machines, this process is not so easy as we humans have to decide if we want to establish norms which we often surface when talking about machine interactions as protocols and best practices or requirements.
```
Comment (justinabrahms) [inline content/en/learn/tools-terminology/_index.md:14]:
```suggestion
Unfortunately for machines, this process is not so easy. We humans have to decide if we want to establish norms which surface when talking about machine interactions as protocols and best practices or requirements.
```

File: content/en/learn/tools-terminology/_index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Continuous Integration (CI) and Continuous Delivery (CD) practitioners have many tools at their disposal but it is often the case that what we call a pipeline in today’s tool of choice is not called the same thing in the tool we use tomorrow. Again, we can within our sphere of influence and interaction adjust for these nuances but machines talking to one another do not have that same luxury necessarily.
```
Comment (justinabrahms) [inline content/en/learn/tools-terminology/_index.md:16]:
```suggestion
Continuous Integration (CI) and Continuous Delivery (CD) practitioners have many tools at their disposal but it is often the case that what we call a pipeline in today’s tool of choice is not called the same thing in the tool we use tomorrow. Again, we can, within our sphere of influence and interaction, adjust for these nuances, but machines talking to one another do not have that same luxury.
```

File: content/en/learn/tools-terminology/_index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+The CDF Interoperability Special Interest Group created a document to collect the basic terms used by CI/CD tools and technologies in order to work on establishing a shared vocabulary for us humans to communicate and collaborate better.
```
Comment (justinabrahms) [inline content/en/learn/tools-terminology/_index.md:18]:
```suggestion
The CDF [Interoperability Special Interest Group](https://github.com/cdfoundation/sig-interoperability/#readme) created a document to collect the basic terms used by CI/CD tools and technologies in order to work on establishing a shared vocabulary for us humans to communicate and collaborate better.
```

**Review by justinabrahms — APPROVED** [review]
Added a bunch of commas and moved a few words around.

**Comment by tdcox** [comment]
Thanks. Please can you put this in the References section, as was proposed previously?

File: content/en/learn/tools-terminology/_index.md (hunk 1 above)
Comment (afrittoli) [reply content/en/learn/tools-terminology/_index.md]: Thanks for the review @justinabrahms - this text is copy-pasted from the interoperability page as a way of introduction - https://github.com/cdfoundation/sig-interoperability/blob/main/docs/tools-terminology.md#terminology-used-by-cicd-tools-and-technologies. I'm happy to update it as you suggested, but shall we update it back in the sig interop page as well?

**Comment by afrittoli** [comment]
I moved the page into the resources section - I did not find "references", but the rendering doesn't look good.
Perhaps I could simply add a link into the resources page and avoid adding a new page?

**Comment by tdcox** [comment]
Ah, it will need a layout for resources to render nested pages. See https://github.com/cdfoundation/best-practices-site/tree/main/themes/docsy/layouts

You should be able to copy one of the others for now.

File: content/en/learn/tools-terminology/_index.md (hunk 1 above)
Comment (justinabrahms) [reply content/en/learn/tools-terminology/_index.md]: Sounds wonderful. :)

**Pull request merged**